scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -324,14 +329,27 @@ def get_pi_executions(mysqlcursor_orca):
         logging.error("Exception occurred while getting the pipeline execution : ", exc_info=True)
         raise e
 
+def get_postgres_pi_executions():
+    try:        
+        spindb_orca_postgres.execute("SELECT pi.application, pi.body, ps.body FROM pipeline_stages ps LEFT OUTER JOIN pipelines pi ON ps.execution_id = pi.id")
+        return spindb_orca_postgres.fetchall()        
+    except Exception as e:
+        print("Exception occurred while getting the pipeline executions : ", e)
+        logging.error("Exception occurred while getting the pipeline execution : ", exc_info=True)
+        raise e
 
 def updated_stage_execution_data(pi_stage_id, updated_json):
     try:
-        dump_updated = json.dumps(updated_json)					
-        mysqlcursor_orca = spindb_orca.cursor()
+        dump_updated = json.dumps(updated_json)
         sql = "UPDATE pipeline_stages SET body = %s WHERE id = %s"
         data = (dump_updated, pi_stage_id)
-        mysqlcursor_orca.execute(sql, data)        
+        if spin_db_type == 'sql':
+           mysqlcursor_orca = spindb_orca_sql.cursor()        
+           mysqlcursor_orca.execute(sql, data)
```

Comment:
shouldn't we close cursor after this execute ? 
@sudhakaropsmx any side effect of closing cursor here ?

Reply:
added to close cursor